External/FEXCore/Source/Interface/Core/OpcodeDispatcher/Flags.cpp

```diff
@@ -52,10 +52,9 @@ void OpDispatchBuilder::SetPackedRFLAG(bool Lower8, OrderedNode *Src) {
     InvalidateDeferredFlags();
   }
 
-  auto OneConst = _Constant(1);
   for (size_t i = 0; i < NumFlags; ++i) {
     const auto FlagOffset = FlagOffsets[i];
-    auto Tmp = _And(_Lshr(Src, _Constant(FlagOffset)), OneConst);
+    auto Tmp = _Bfe(4, 1, FlagOffset, Src);
```

Comment:
This one looks good, assuming the guest's compiler can be trusted to use BEXTR or something.

Reply:
This IR goes through our backend, so usually 1:1 relates to instructions (since the IR optimization are light).
So this causes a `lsr + and + ubfx + strb` per flag to convert in to a `ubfx + strb`. Cutting the number of uops in half and the number of data dependencies in to far less.

Reply:
right, makes sense :+1: